_docs/documenting_rest_api_endpoints/docapis_new_endpoint_to_doc.md

```diff
@@ -64,10 +64,10 @@ https://simple-weather.p.mashape.com/surfreport/123?&days=2&units=metrics&hour=1
 <p>surfreport: </p>
 
 <ul>
-<li>surfheight (time: feet)</li>
-<li>wind (time: kts)</li>
-<li>tide (time: feet)</li>
-<li>water temperature (time: F degrees)</li>
+<li>surfheight (units: feet)</li>
```

Comment:
I made a guess here; `time` didn't feel right. Did you mean `units` or did I get it wrong?